src/modules/cpu/host_tensor_augmentations.hpp

```diff
@@ -5265,4 +5265,549 @@ RppStatus color_cast_i8_i8_host_tensor(Rpp8s *srcPtr,
+                dstPtrRowR += dstDescPtr->strides.hStride;
+                dstPtrRowG += dstDescPtr->strides.hStride;
+                dstPtrRowB += dstDescPtr->strides.hStride;
+            }
+
+            for(int i = 0; i < roiPtr3->xywhROI.roiHeight; i++)
+            {
+                Rpp8u *srcPtrTemp3, *srcPtrTemp4, *dstPtrTempR, *dstPtrTempG, *dstPtrTempB;
+                srcPtrTemp3 = srcPtrRow3;
+                srcPtrTemp4 = srcPtrRow4;
+                
+                dstPtrTempR = dstPtrRowR;
+                dstPtrTempG = dstPtrRowG;
+                dstPtrTempB = dstPtrRowB;
+
+                int vectorLoopCount3 = 0;
+                for (; vectorLoopCount3 < alignedLength3; vectorLoopCount3+=48)
+                {
+                    __m128 p[3];
+
+                    rpp_simd_load(rpp_load48_u8pkd3_to_u8pln3, srcPtrTemp3, p);    // simd loads
+                    rpp_simd_store(rpp_store48_u8_to_u8, dstPtrTempR, dstPtrTempG, dstPtrTempB, p);    // simd stores
+
+                    srcPtrTemp3 += 48;
+                    dstPtrTempR += 16;
+                    dstPtrTempG += 16;
+                    dstPtrTempB += 16;
+                }
+                for (; vectorLoopCount3 < bufferLength3; vectorLoopCount3+=3)
+                {
+                    *dstPtrTempR = srcPtrTemp3[0];
+                    *dstPtrTempG = srcPtrTemp3[1];
+                    *dstPtrTempB = srcPtrTemp3[2];
+
+                    srcPtrTemp3 += 3;
+                    dstPtrTempR++;
+                    dstPtrTempG++;
+                    dstPtrTempB++;
+                }
+
+                int vectorLoopCount4 = 0;
+                for (; vectorLoopCount4 < alignedLength4; vectorLoopCount4+=48)
+                {
+                    __m128 p[3];
+
+                    rpp_simd_load(rpp_load48_u8pkd3_to_u8pln3, srcPtrTemp4, p);    // simd loads
+                    rpp_simd_store(rpp_store48_u8_to_u8, dstPtrTempR, dstPtrTempG, dstPtrTempB, p);    // simd stores
+
+                    srcPtrTemp4 += 48;
+                    dstPtrTempR += 16;
+                    dstPtrTempG += 16;
+                    dstPtrTempB += 16;
+                }
+                for (; vectorLoopCount4 < bufferLength4; vectorLoopCount4+=3)
+                {
+                    *dstPtrTempR = srcPtrTemp4[0];
+                    *dstPtrTempG = srcPtrTemp4[1];
+                    *dstPtrTempB = srcPtrTemp4[2];
+
+                    srcPtrTemp4 += 3;
+                    dstPtrTempR++;
+                    dstPtrTempG++;
+                    dstPtrTempB++;
+                }
+
+                srcPtrRow3 += srcDescPtr->strides.hStride;
+                srcPtrRow4 += srcDescPtr->strides.hStride;
+                dstPtrRowR += dstDescPtr->strides.hStride;
+                dstPtrRowG += dstDescPtr->strides.hStride;
+                dstPtrRowB += dstDescPtr->strides.hStride;
+            }
+        }
+        
```

Comment:
Enable settings -> remove trailing whitespace while saving file on VScode. And also, remove this additional line

Reply:
Done